addons/account/wizard/account_move_reversal.py

```diff
@@ -98,6 +98,7 @@ def reverse_moves(self):
             'name': _('Reverse Moves'),
             'type': 'ir.actions.act_window',
             'res_model': 'account.move',
+            'context': {'default_type': 'out_refund'},
```

Comment:
@FlorianGilbert  Roger that, changes done. I didn't use the default_values_list because this dic hasn't got the type key/value.

Reply:
@PabloPaezSheridan Your changes are not correct. May be it works but that's not really clean. You can't just take the last move type because there could be more than one type of account move (in case of you try to do a reverse in JE views).
You can maybe doing something like that: 
```python
moves_type = moves.mapped('type')
move_type = len(set(moves_type)) == 1 and moves_type.pop() or False

...

action = {
    'name': _('Reverse Moves'),
    'type': 'ir.actions.act_window',
    'res_model': 'account.move',
}

if move_type:
    action['context'] = {'default_type': move_type'}
```
May be it's better to just delete this part and don't set any default type.

Reply:
Well @FlorianGilbert  I've seen this with a more expert partner and i have made the changes you ask for. I have used moves_to_redirect because it has the reversed moves instead of the moves recordset, it just a detail.